internal/operands/metrics/resources.go

```diff
@@ -99,9 +101,10 @@ var alertRulesList = []promv1.Rule{
 			"runbook_url": runbookURLBasePath + "SSPTemplateValidatorDown",
 		},
 		Labels: map[string]string{
-			severityAlertLabelKey:  "critical",
-			partOfAlertLabelKey:    partOfAlertLabelValue,
-			componentAlertLabelKey: componentAlertLabelValue,
+			severityAlertLabelKey:     "critical",
+			healthImpactAlertLabelKey: "critical",
```

Comment:
@ksimon1 @akrejcir should this alert have a critical severity?  Would a template validator require waking someone in the middle if the night?

Reply:
Based on the runboks, https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/SSPTemplateValidatorDown.md , I would argue that the severity is warning. But the health impact is critical. What do you think?

Reply:
AFAIR when template validator is down, it will prevent creation of any VM from template.

Reply:
Who can answer this? Is this indeed critical on the cluster level and require handling immediately?

Reply:
Template validator is a validating webhook, that gets called for every VM create or update. 
If it is down, then no VMs can be created or updated, which also means a VM cannot be started.

Reply:
@akrejcir is it critical on the cluster level ? or only on the operator health level?

Reply:
I'm not sure. What is the difference between the levels?

Reply:
Thanks. According to this part of the docs: 

> Reserve critical level alerts only for reporting conditions that may lead to loss of data or inability to deliver service for the cluster as a whole.

the severity does not need to be critical. Without the Template Validator, VMs cannot be created and started, but other cluster functionality works.

Reply:
@akrejcir  I was just wondering about this part, "inability to deliver service for the cluster as a whole." 
Is creating and updating VMs considered part of the must have for every cluster with CNV installed?
I think the answer is No, but I want to make sure..

Reply:
The same reason can probably be applied to other alerts in SSP.

Reply:
I don't know the answer.

Reply:
I consulted with Ronen. Lets ignore this for this PR. 
I will open a different story/bug in case we need to update the severity for these alerts.